main.go

```diff
@@ -63,13 +62,7 @@ func handleJobs(jobsC <-chan context.Job) {
 func executeTasks(tasks []*context.Task, watcher *context.Watcher) {
 	for _, task := range tasks {
 		command := task.Command
-		tokens := strings.Split(command, " ")
-		name := tokens[0]
-		var cmdArg []string
-		if len(tokens) > 1 {
-			cmdArg = tokens[1:]
-		}
-		cmd := exec.Command(name, cmdArg...)
+		cmd := exec.Command("/bin/bash", "-c", command)
```

Comment:
I've discovered a package for exactly this use-case: https://github.com/mattn/go-shellwords

It supports env vars, subcommands (that are run in SHELL but in a cross-platform way), quotes. The change required for its use is:
```diff
-		tokens := strings.Split(command, " ")
+		tokens, err := shellwords.Parse(command)
+		if err != nil {
+			...
+ 		}
		name := tokens[0]
		var cmdArg []string
		if len(tokens) > 1 {
			cmdArg = tokens[1:]
		}
		cmd := exec.Command(name, cmdArg...)
```